src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
1. Error handling: 
In the `throwError` method, it's good to be explicit about the exceptions that could be thrown. You could use a more specific exception type instead of a generic `RuntimeException`. Consider creating a custom exception class for specific types of errors that can occur in your application. This will make it easier to handle exceptions and provide better error messages to users.